src/main/java/net/fabricmc/loom/LoomGradleExtension.java

```diff
@@ -426,4 +432,20 @@ public synchronized File getNextMixinMappings() {
 	public List<LoomDecompiler> getDecompilers() {
 		return decompilers;
 	}
+
+	@ApiStatus.Experimental
+	public void runs(Closure<?> conf) {
```

Comment:
I'd just use an `Action<NamedDomainObjectContainer<RunConfigSettings>>` here too. That should work nicely in both kt and groovy dsl.

Reply:
Seems to work fine in groovy 👍